tests/system/providers/google/cloud/cloud_functions/example_functions.py

```diff
@@ -19,25 +19,6 @@
 """
 Example Airflow DAG that displays interactions with Google Cloud Functions.
 It creates a function and then deletes it.
-
-This DAG relies on the following OS environment variables
-https://airflow.apache.org/concepts.html#variables
-
-* PROJECT_ID - Google Cloud Project to use for the Cloud Function.
```

Comment:
As this is still loaded from the env var, should we keep this one?

Reply:
It's common accross all system tests I think the comments here were a bit unnecessary.